examples/stm32/f0/other/can/Makefile

```diff
@@ -0,0 +1,24 @@
+##
+## This file is part of the libopencm3 project.
+##
+## Copyright (C) 2009 Uwe Hermann <uwe@hermann-uwe.de>
```

Comment:
nope. If you believe this is trivial enough, simply omit the line.  But Uwe did not write this file.